### DISCUSSION TITLE
blogs – Data Down To Earth

### DISCUSSION THREAD

**Discussion by giscus** [quarto-dev/quarto-web#1243] [General]
# blogs – Data Down To Earth

http://localhost:63342/DataDownToEarth/docs/blogs/blogs.html

**Comment by iamravis** [comment]


**Reply by cderv** [reply]
Hi @iamravis 

Be sure to configure correctly your website with commenting using giscus because it seems you dev website is right now configured to post in quarto-web. Thanks
